src/status_im/contexts/wallet/common/utils.cljs

```diff
@@ -315,3 +315,26 @@
 
     :else
     constants/mainnet-chain-ids))
+
+(defn total-owned-collectible
+  [ownership]
+  (reduce (fn [acc item] (+ acc (js/parseInt (:balance item)))) 0 ownership))
+
+(defn total-owned-collectible-by-address
+  [ownership address]
+  (reduce (fn [acc item]
+            (if (= (:address item) address)
+              (+ acc (js/parseInt (:balance item)))
+              acc))
+          0
+          ownership))
+
+(defn calculate-owned-collectible
+  [ownership address]
+  (if address
+    (total-owned-collectible-by-address ownership address)
+    (total-owned-collectible ownership)))
+
+(defn collectible-owned-counter
+  [total]
+  (when (> total 1) total))
```

Comment:
This fn doesn't seem as a counter :thinking: maybe better to rename it

Reply:
I think a counter counts something, but this seems like an identity function with a constraint to return `nil`.

I wonder if we really need this as a `utils` function, how many usages does it have? and why do we need a counter returning `nil` when the given number is `1` or `0`?

Maybe we want `nil` for some specific cases, e.g. if there's more than `1` we'll show a listing, in that case we can name it `show-collectible-listing?` and move it to its namespace instead of exposing it in utils.

Could you please elaborate more on the use cases of this counter?

Thanks @mmilad75 !

Reply:
@ulisesmac It's been used in two components, `collectible-list-item` and `expanded-collectible`. We want to show the number of a single collectible owned by the user. If it's more than one, then we show a label on the collectible. This function calculates this. If the collectible is owned more than 1, then return the result, and if it's 1, then return nil.

Reply:
I updated the code as I had missed to add `x` to the beginning of the number

Reply:
I think this function doesn't need to be in `common.utils` could be just a check inline. wdyt?

cc: @J-Son89 IMO this is not a general utility fn, we were discussing this recently, wdyt?

Reply:
yeah probably better that we do that inline? perhaps we can even put this inside the quo component instead? 🤔